intro_to_r/intro_to_r.Rmd

```diff
@@ -239,9 +251,21 @@ of operations.  Here, we want to divide the number of boys by the total number
 of newborns, so we have to use parentheses.  Without them, R will first do the 
 division, then the addition, giving you something that is not a proportion.
 
+We can also pipe data into plots, as in
+```{r piped-plot}
+arbuthnot %>%
```

Comment:
I think it's reasonable to call `.` a pronoun. 

I find the combination of `qplot()` and very jarring. I suspect it might be harder for new R users to understand pipes when the object undergoes a radical transformation during the pipe (as here, where it goes from data to plot).

Pipes do work a bit more elegantly if you use `ggplot()`:

```R
arbuthnot %>%
  mutate(prop = boys / (boys + girls)) %>%
  ggplot(aes(year, prop)) + 
    geom_line()
```

Although the transition from `%>%` to `+` is still awkward